environments/rinkeby/permissions.yml

```diff
@@ -51,6 +51,37 @@
       - 'CREATE_VERSION_ROLE'
     - ppf-factory.aragonpm.eth:
       - 'CREATE_VERSION_ROLE'
+- address: '0x9b1B224E0445243eF5fD102114d15136967FfB15'
+  owner: Aragon Black testnet cold wallet
```

Comment:
@osarrouy Just wanted to double check that this is a cold wallet?

Reply:
Yes this is!